client/shipping-label/state/actions.js

```diff
@@ -467,14 +467,17 @@ export const purchaseLabel = () => ( dispatch, getState, context ) => {
 			return;
 		}
 		const state = getState().shippingLabel;
+		const packageParamBlacklist = [  'items', 'id', 'box_id' ];
+		const templateParamBlacklist = [ ...packageParamBlacklist, 'width', 'length', 'height' ];
```

Comment:
This seems backwards compared to the way package and template whitelists are defined above?